android-xbuild-target/README.md

```diff
@@ -0,0 +1,3 @@
+# android-xbuild-target
+
+This is a crate that can be built via `x build` in order to simplify CI testing.
```

Comment:
Maybe we should create a common `xbuild` dir and put these 2 crates there with common docs.